src/index.js

```diff
@@ -42,6 +42,21 @@ function getAllAliases(options) {
 	return get(options, 'aliases', {});
 }
 
+/**
+ * Runs an array of promises in series
+ *
+ * @method pSeries
+ *
+ * @param  {Array} items
+ * @param  {Function} iterator
+ * @return {Promise}
+ */
+function pSeries(items, iterator) {
```

Comment:
Let's call it `promiseSeries`, and we're done here ;-)

Reply:
Sure!